views/page/history.php

```diff
@@ -38,6 +42,9 @@
             <div class="wiki-headline">
                 <div class="wiki-headline-top">
                     <?= WikiPath::widget(['page' => $page]) ?>
+                    <?php if($page->latestRevision->revision_label != null): ?>
+                        <?= Button::info($hideMinorChanges ? Yii::t('WikiModule.base', 'Show minor changes') : Yii::t('WikiModule.base', 'Hide minor changes'))->sm()->link(Url::current(['toggle-hide-minor-changes']))->cssClass('Button-right-align') ?>
```

Comment:
Please add some right margin/padding to the button:
<img width="415" height="80" alt="image" src="https://github.com/user-attachments/assets/9ab6e424-92f9-4401-9701-c0cf41323072" />